api/v1alpha1/databasecluster_types.go

```diff
@@ -33,6 +33,8 @@ var (
 const (
 	// AppStateUnknown is an unknown state.
 	AppStateUnknown AppState = "unknown"
```

Comment:
should we remove this since it is not used anywhere now?

Reply:
in addition - is AppStateNew used somehow? and shall it be replaced with AppStateCreating as well?

Reply:
The way we map statuses now is `everestv1alpha1.AppState(upstream.Status.State)` which means we still need both 

AppStateUnknown AppState = "unknown"
AppStateNew AppState = ""

because pxc has "unknown", psmdb has "", pg does not have any empty status.

As an alternative we could remove the statuses^ and have a `switch` mapping in each provider separately. I thought it's not really needed right now because the other statuses are mapping 1:1. Do you think it would be better to have the mapping for each upstream operator? I agree it's a bit confusing to have the statuses that are used only internally, but not sure which option would be better here.

Reply:
> have a switch mapping in each provider separately

I can't remember clearly, but I think we used to have this earlier and we removed it for some reason.

But anyway, I'm fine with keeping them around since they are used in upstream operators. My concern was that these vars are not used anywhere currently, but its probably fine to keep them